addons/mail/static/src/composer/composer.xml

```diff
@@ -23,7 +23,7 @@
                 <span class="cursor-pointer" t-on-click="() => env.messageHighlight?.highlightMessage(props.messageToReplyTo.message.id, props.composer.thread)">
                     Replying to <b t-esc="props.messageToReplyTo.message.author.name"/>
                 </span>
-                <span t-if="props.messageToReplyTo.message.originThread !== thread">
+                <span t-if="props.messageToReplyTo.message.originThread !== props.thread">
```

Comment:
I don't see this prop, isn'it it `props.composer.thread`?

Reply:
Fixed!